src/explore-education-statistics-admin/src/modules/chart-builder/ChartConfiguration.tsx

```diff
@@ -81,11 +83,10 @@ const InfographicChartOptions = ({
   onChange,
   handleApiErrors,
 }: InfographicChartOptionsProps & ErrorControlProps) => {
-  const [chartFileOptions, setChartFileOptions] = React.useState<
-    SelectOption[]
-  >([]);
+  const [chartFileOptions, setChartFileOptions] = useState<SelectOption[]>([]);
 
-  const [uploading, setUploading] = React.useState(false);
+  const [uploading, setUploading] = useState(false);
+  const [deleteFile, setDeleteFile] = useToggle(false);
```

Comment:
When using `useToggle`, you should name the setter `toggleDeleteFile`. Then you can use that directly in callback props i.e. `onExit={toggleDeleteFile}`